### ISSUE TITLE
DDW Fact

### ISSUE THREAD

**Issue by Napho** [devinit/datahub#340]
Move data from mirror databases to fact tables. Fact table holds aggregated data values from which other schemas that directly relate to ddw UI get pick their data values

**Comment by Napho** [comment]
Non-dac deflators needed to generate the final master deflator table for use in the ddh update.
Deflators are from DAC1, DAC2A, DAC2b and DAC5 mirrors, if the deflator cannot be found, in any of the above then IMF deflators should be used, if IMF cannot be used then World Bank has to be used

**Comment by k8hughes** [comment]
@Napho @Duncan-Knox can you add more detail here following your meeting?

**Comment by Duncan-Knox** [comment]
We are looking into the deflator requirements for the fact data series.

**Comment by k8hughes** [comment]
It seems that there are some things that don't need deflating, should we split this and have a separate ticket for the population stuff so that part can be completed and passed to the front end?

**Comment by Napho** [comment]
@k8hughes  that is the plan now, we are trying to single out tables that don't need any deflation to be worked on first in this sprint, then save the heavier work for the later part of the sprint

**Comment by k8hughes** [comment]
Is this issue now able to be closed?
